readme.md

```diff
@@ -116,6 +116,16 @@ final class PreferencesViewController: NSViewController {
 }
 ```
 
+## Localization
+
+This package supports localization but still in early development, so PRs are welcome!
+
+1. Fork the repo.
+2. Create a directory that has a name that uses an ISO 639 language code and optional designators, followed by the `.lproj` suffix, and doesn’t contain subdirectories, you can find more [here](https://developer.apple.com/documentation/swift_packages/localizing_package_resources).
+3. Create a file named `Localizable.strings` under the new language directory, and copy all text from `KeyboardShortcuts/Localization/en.lproj/Localizable.strings` to the new file that you created.
+4. Localize!
+5. Submit a PR.
```

Comment:
Can you add something like this as step 6: Try to find someone that speaks your language to review the translation.